modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c

```diff
@@ -0,0 +1,524 @@
+/*
+ * PROJECT:     ReactOS API Tests
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Unit Tests for acpi!Bus_PDO_QueryResourceRequirements
+ * COPYRIGHT:   Copyright 2017-2020 Thomas Faber (thomas.faber@reactos.org)
+ */
+
+#include <apitest.h>
+
+#define WIN32_NO_STATUS
+#include <ndk/rtlfuncs.h>
+#define UNIT_TEST
+#include <acpi.h>
+
+/* Kernel definitions (copied) */
```

Comment:
It's not possible to include a header there?

Reply:
The header would be wdm.h, this is user mode code. So... probably not.